### PR TITLE
Fix Rack Recipe

### PR DESCRIPTION
**Pull request by pmardle**
Fixes Rack Recipe from using logs to planks.

# Changes proposed in this pull request:
Remove requirement for Wood Logs from recipe and add in Wood_Planks - as per the original intent :)
-
-

Review please

### CODE REVIEW

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 29 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 27 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecipeHandler.java#L94](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L94): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L113](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L113): Remove this unused private "addHutRecipe" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [entity](https://github.com/Minecolonies/minecolonies/blob/503bdec8821559db6b1cc49a7669352eda4fb3ee/src/main/java/com/minecolonies/coremod/entity): Add a 'package-info.java' file to document the 'entity' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 1]
```diff
@@ -81,8 +81,8 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
                 "XXX", "X#X", "XXX", 'X', "logWood", '#', ModItems.scanTool));
         //Block Rack
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockRack, 1),
-                "XXX", "X#X", "XXX", 'X', "logWood", '#', Blocks.IRON_BARS));
-
+                "XXX", "X#X", "XXX", 'X', "PLANK_WOOD", '#', Blocks.IRON_BARS));
+            
         //Field
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutField, 1), 
                 " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
```
Comment (marchermans) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java]: Please fix this too it is : stick_wood.

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 2]
```diff
@@ -81,8 +81,8 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
                 "XXX", "X#X", "XXX", 'X', "logWood", '#', ModItems.scanTool));
         //Block Rack
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockRack, 1),
-                "XXX", "X#X", "XXX", 'X', "logWood", '#', Blocks.IRON_BARS));
-
+                "XXX", "X#X", "XXX", 'X', "PLANK_WOOD", '#', Blocks.IRON_BARS));
```
Comment (marchermans) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java]: Lowercase OreDic Names only!

**Review by marchermans — CHANGES_REQUESTED** [review]
Some things are not correct.

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java (hunk 1 above)
Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/util/RecipeHandler.java]: it is correct, WOODEN_STICK is a static final var which has stick_wood

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java (hunk 2 above)
Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/util/RecipeHandler.java]: You only need to remove the "" here

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1279/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 29 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 27 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecipeHandler.java#L94](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L94): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [RecipeHandler.java#L113](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L113): Remove this unused private "addHutRecipe" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [entity](https://github.com/Minecolonies/minecolonies/blob/7580cb82ea9ab9f4500203b91089ca3f10ed4498/src/main/java/com/minecolonies/coremod/entity): Add a 'package-info.java' file to document the 'entity' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java (hunk 1 above)
Comment (marchermans) [reply src/main/java/com/minecolonies/coremod/util/RecipeHandler.java]: Oeps missed that, lol 😆

**Review by marchermans — APPROVED** [review]

**Pull request merged**